app/pb_kits/playbook/pb_selectable_list/docs/_selectable_list_checkbox.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= pb_rails("selectable_list",
+        props: {
+          variant: "checkbox",
+          items: [
+            { text: "one", name: "custum1" },
+            { text: "two", name: "custum2" },
+            { text: "three", name: "custum3" },
+            { text: "four", name: "custum4" },
+          ]
+        }
+    )
+%>
```

Comment:
@jasoncypret We usually have the item as their own kit. This works, but it's not consistent with how the other kit works. Should we continue with this?

Reply:
We should. We worked hard at making this invoke the child kit. But Ruby won't behave the same way JS/React will here.  @andrew-k9 - This PR was supposed to contain lots of notes about this effort. Can you please elaborate on why we had to go with this design?

Reply:
Oh wait. It does!! @christinaatai - Check out the "What we tried" section in the summary. :)

Reply:
Another thing to keep in mind is many of the form related kits have an option to bring your own input. This is because when using these with simple form it's best to just wrap the input in the kit. Just another potential reason to build items as sub kits.

Reply:
FWIW this is the only way to add `selectable_list_item`s to the `selectable_list`. We tried an approach like 

```ruby
<%= pb_rails("selectable_list", props: { variant: "checkbox" }) do %>
  <%= pb_rails("selectable_list/selectable_list_item", props: { text: "Unselected", name: "custum1" }) %>
        props: {
  <%= pb_rails("selectable_list/selectable_list_item", props: {text: "Selected", name: "custum1"}) %>
          variant: "radio",
  <%= pb_rails("selectable_list/selectable_list_item", props: {text: "Hover", name: "custum1"}) %>
          items: [
  <%= pb_rails("selectable_list/selectable_list_item", props: {text: "Unslected Last", name: "custum1"}) %>
            { text: "one", name: "custum1" },
<% end %>
```

before and it does not allow us to add the `variant` prop to `selectable_list`s children. In 
```ruby
capture(&object.children)
```
we would first need to take each child element, give it a `variant` prop, and then have `capture` present the block to the `selectable_list_item.html.erb` file (unsure of wording for that 🤔) which I don't think is possible with procs in ruby without doing something like 

```ruby
pb_rails("selectable_list") do |variant|
```
when being called 😳